Code/Editor/EditorPreferencesDialog.cpp

```diff
@@ -112,6 +112,22 @@ void EditorPreferencesDialog::showEvent(QShowEvent* event)
     QDialog::showEvent(event);
 }
 
+
+void EditorPreferencesDialog::keyPressEvent(QKeyEvent* event)
```

Comment:
nit: line 124 the formatting of the if is a bit off (should be `if (QWidget*...`) - if you select this block and hit Ctrl-K, F then clang-format will do its thing and auto-format it all

Reply:
Re-formatted.